### DISCUSSION TITLE
Transaction fees associated with createNft

### DISCUSSION THREAD

**Discussion by katewayo** [metaplex-foundation/js#81] [Q&A]
Hi @lorisleiva! Hope all is well. Is there a way to expose the transaction fee associated with the full createNft operation in the output along with the Nft? This would be useful for composing the createNft operation with other operations where the feePayer changes (and you're trying to recover the minting fees along with other relevant charges). Does that make sense?

**Comment by lorisleiva** [comment]
Hi there 👋 

Do you mean a way to use the SDK to estimate the "transaction fees" and "storage fees" associated with creating an NFT?

If so, not at the moment but this is something I'd like the SDK to support at some point for some of the low-level patterns it uses (such as transaction builders).

In the meantime, you'd just need to gather the size of all accounts created (e.g. token account, mint account, metadata account, master edition account, etc.) and use `connection. getMinimumBalanceForRentExemption` to figure out how much storage fees will be needed. You'll also need to add the transaction fee of each transaction you send but these are usually significantly lower than storage fees anyway.

**Comment by katewayo** [comment]
Yes, I do mean estimating the transaction and storage fees associated with creating an NFT. Great to hear that it's something you'd like to support as it will help with the way I'm composing on Metaplex. In the meantime, I'll use the storage fee estimation you suggest as a proxy. Thanks!
